package com.xworkz.exception;

import java.awt.*;

public class Block
{
    public static void main(String[] args)
    {
        System.out.println("main started");
        Block block=new Block();
        System.out.println("main ended");

    }
    {
        System.out.println("invoking first init block");
    }
    {
        System.out.println("invoking second init block");
    }
    {
        System.out.println("invoking third init block");
    }
    {
        System.out.println("invoking fourth init block");
    }
    {
        System.out.println("invoking fifth init block");

    }
}
